apps/repo_poller/test/setup_worker_test.exs

```diff
@@ -74,6 +74,7 @@ defmodule RepoPoller.SetupWorkerTest do
              end)
 
     worker_pid = Process.whereis(:f@k3)
+    :timer.sleep 200
```

Comment:
hmmm, we need to investigate why we need a timer here if we are already "active waiting" above for workers to be created